bullet_train/lib/bullet_train/resolver.rb

```diff
@@ -26,9 +26,8 @@ def run(eject: false, open: false, force: false, interactive: false)
         puts ""
 
         if interactive && !eject
-          puts "\nWould you like to eject the file into the local project? (y/n)\n"
+          print "\nWould you like to eject the file into the local project? (y/n): "
```

Comment:
Unless there's a specific reason for using `print` over `puts` here, I think we should keep it as `puts` because that's what we use across the board for a lot of our other scripts (i.e. - [bin/configure](https://github.com/bullet-train-co/bullet_train/blob/main/bin/configure)).

If we want to change something stylistic like this I think it should have its own PR and rationale written there.

Reply:
I actually personally like the one-line setup you proposed:
```
> Would you like to eject the file into the local project? (y/n):  y
```

Also, totally valid about the `ask` method. We should be consistent with that too. Hopefully we can open up another PR or an issue addressing that soon!